hedera-node/hedera-config/src/main/java/com/hedera/node/config/converter/PermissionedAccountsRangeConverter.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.config.converter;
+
+import com.hedera.node.app.service.mono.context.domain.security.PermissionedAccountsRange;
+import com.swirlds.config.api.converter.ConfigConverter;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+
+public class PermissionedAccountsRangeConverter implements ConfigConverter<PermissionedAccountsRange> {
```

Comment:
Please add some class comments, which should include the specification of what exactly is the supported text format (spaces, no spaces, negatives, etc).

Also consider making the class final if it is not intended to be overridden (although in this case it seems harmless).